website/src/content/docs/linter/linter-sources.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Linter sources
+description: A page that maps lint rules from other sources to Biome
+---
+    
```

Comment:
We could add a warning saying that most of the rules doesn't provide the equivalent options provided by ESlint?